### PR TITLE
header logo resolution and alignment:

### PR DESCRIPTION
**Pull request by ksiegel1923**
changed header logo from a png to an svg (help with resolution)
added extra margin and padding to header logo so visually it appears centered

### CODE REVIEW

File: feedback-module/src/assets/styling_classnames.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const CHECKBOX_STYLE =
   "text-bold text-primary bg-transparent mobile-lg:font-sans-lg mobile-lg-font-sans-lg font-sans-md mobile-lg:margin-y-2 mobile-lg-margin-y-2 margin-y-1 text-no-uppercase";
 
 export const HEADER_STYLE =
-  "radius-top-lg bg-primary display-flex padding-1 mobile-lg:padding-x-2 mobile-lg:padding-y-105 mobile-lg-padding-x-2 mobile-lg-padding-y-105 font-sans-check";
+  "radius-top-lg bg-primary display-flex padding-top-105 padding-bottom-1 padding-x-1 mobile-lg:padding-x-2 mobile-lg:padding-bottom-105 mobile-lg:padding-top-205 mobile-lg-padding-x-2 mobile-lg-padding-top-2 mobile-lg-padding-bottom-105 font-sans-check";
```
Comment (michellexliu) [inline feedback-module/src/assets/styling_classnames.js:9]: note: should probably wait until other PR is merged in and move this to sass

File: feedback-module/src/components/common/Header.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ function Header({ innerRef }) {
       <div ref={innerRef}></div>
       <GridContainer className={HEADER_STYLE}>
         <img
-          className="width-auto height-2 mobile-lg:height-205 mobile-lg-height-205"
+          className="width-auto height-2 mobile-lg:height-205 mobile-lg-height-205 margin-top-1px"
```
Comment (michellexliu) [inline feedback-module/src/components/common/Header.js:14]: note: also need to move this

File: feedback-module/src/assets/styling_classnames.js (hunk 1 above)
Comment (ksiegel1923) [reply feedback-module/src/assets/styling_classnames.js:9]: yeah definitely just left this here as a placeholder so we know what styles to implement

Comment (michellexliu) [reply feedback-module/src/assets/styling_classnames.js:9]: should it be `mobile-lg:padding-top-205` or `mobile-lg:padding-top-2`? I noticed that you had `mobile-lg:padding-top-205` first then `mobile-lg-padding-top-2` later on.

Comment (ksiegel1923) [reply feedback-module/src/assets/styling_classnames.js:9]: mobile-lg:padding-top-2

**Pull request merged**